index.js

```diff
@@ -105,7 +144,15 @@ function findById() {}
  *  filterByGenre(movies, "Horror")
  *  //> []
  */
-function filterByGenre() {}
+function filterByGenre(movies,genre) {
+  let movieGenre = []
+  for(let i = 0; i < movies.length; i++){
+  if ( movies[i].genre === genre ){
```

Comment:
Did you take a look at what the genre is in the `movie` object? Looks like it is a comma separated string of title case genres. Likely not equal to the passed in `genre.`